chains/solana/contracts/programs/fee-quoter/src/context.rs

```diff
@@ -75,6 +75,23 @@ pub struct UpdateConfig<'info> {
     pub authority: Signer<'info>,
 }
 
+#[derive(Accounts)]
+pub struct UpdateConfigLinkMint<'info> {
+    #[account(
+        mut,
+        seeds = [seed::CONFIG],
+        bump,
+        constraint = valid_version(config.version, MAX_CONFIG_V) @ FeeQuoterError::InvalidVersion,
+    )]
+    pub config: Account<'info, Config>,
+
+    pub link_token_mint: InterfaceAccount<'info, Mint>,
```

Comment:
I'm passing the new mint via an account (so using a new context struct) and not just instruction data so that we can look at the decimals configured in the mint